src/biogeochem/CNCIsoAtmTimeSeriesReadMod.F90

```diff
@@ -142,6 +167,12 @@ subroutine C14_init_BombSpike()
     allocate(atm_delta_c14(nsectors_c14,ntim))
     atm_delta_c14(:,:) = 0.0_r8
 
+    ! Allocate the gridcell arrays
+    allocate(atm_delta_c14_grc(bounds%begg:bounds%endg))
```

Comment:
With allocates, there really should be a clean subroutine to do the deallocation. But, since we don't have a place it would be called, this doesn't seem helpful right now.

Reply:
Documented under #3568 so resolving.